src/server/pkg/watch/watch.go

```diff
@@ -1,6 +1,6 @@
 // Package watch implements better watch semantics on top of etcd.
 // See this issue for the reasoning behind the package:
-// https://github.com/coreos/etcd/issues/7362
```

Comment:
The new link doesn't work, should probably revert this.

Reply:
Oh lol, that's a find and replace error, good catch.